packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -217,24 +217,6 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
     }
   }
 
-  // Check that the sorted order has the _artist_pick track as the first
-  updateOrderArtistPickCheck = (tracks: Array<{ track_id: ID }>) => {
```

Comment:
This is not referenced anywhere and duplicates the logic in `getTracks` in `packages/web/src/common/store/pages/profile/lineups/tracks/sagas.js` so removed it.